c/tests/test_stats.c

```diff
@@ -1346,20 +1346,20 @@ test_paper_ex_relatedness(void)
     tsk_treeseq_from_text(&ts, 10, paper_ex_nodes, paper_ex_edges, NULL, paper_ex_sites,
         paper_ex_mutations, paper_ex_individuals, NULL, 0);
 
-    ret = tsk_treeseq_relatedness(&ts, 2, sample_set_sizes, samples, 1, set_indexes, 0,
-        NULL, &result, TSK_STAT_SITE);
+    ret = tsk_treeseq_genetic_relatedness(&ts, 2, sample_set_sizes, samples, 1,
+        set_indexes, 0, NULL, &result, TSK_STAT_SITE);
     CU_ASSERT_EQUAL_FATAL(ret, 0);
```

Comment:
It would be nice to check the value of ``result`` here also. Can you put in an assert ``CU_ASSERT_DOUBLE_EQUAL`` here with the required value (fine to printf this out first to get the value - we just want to catch problems early if there's ever any regressions).

Reply:
Yep, no problem. Should I do this as part of this PR or add as part of the tests issue?

Reply:
Either, it's up to you. Just let me know.

Reply:
I'll do it with the tests!

Reply:
OK, merging.